src/SwiftBindings/src/SwiftBindings.csproj

```diff
@@ -2,9 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+    <Nullable>disable</Nullable>
   </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../SwiftReflector/SwiftReflector.csproj" />
+    <ProjectReference Include="../../SyntaxDynamo/SyntaxDynamo.csproj" />
+    <ProjectReference Include="../../SwiftRuntimeLibrary/SwiftRuntimeLibrary.csproj" />
 
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
```

Comment:
Use `System.CommandLine` to get a consistent CLI UX with the rest of the .NET product?

Reply:
Thanks, updated.